1-estrategia/README.md

```diff
@@ -1,23 +1,162 @@
+
+Por otro lado, se ha observado que los géneros que crean menos interés con una media de un 1 son el flamenco y el country, por lo que puede que se descarte ofrecer información sobre estos estilos musicales. En cuanto al resto de los géneros, es decir, el jazz, punk, rap, techno y la música clásica, han obtenido una media de 3 puntos sobre 5. Esto quiere decir que aunque estos géneros musicales no sean los preferidos de nuestro público objetivo, tiene un interés en obtener información acerca de ellos. También se les ha ofrecido a los encuestados la posibilidad de indicar algún otro género musical, pero no ha habido respuesta alguna. Por esta razón, descartamos ofrecer información acerca de cualquier género que no se haya nombrado.
+
+**Interés sobre las tendencias o nuevos temas de la actualidad**
+
+En general, todos los encuestados han mostrado interés por saber cuales son las tendencias o nuevos temas musicales de la actualidad. También han mostrado interés en saber más acerca de las canciones que escuchan ellos mismos. El 60% de los encuestados tiene interés en profundizar acerca del autor y la repercusión que está teniendo la canción. Algunos también se han interesado por saber el significado de la canción (40%) o la letra (20%). Viendo estos datos, creemos conveniente ofrecer por un lado, listas de las canciones más reproducidas y por otro, información más precisa sobre las canciones que consumirán nuestros usuarios.
+
+**Modo de obtener información sobre los conciertos/festivales**
+
+También hemos preguntado acerca del modo de obtener información sobre los conciertos y festivales. En este caso, han sido tres los que mayor porcentaje han logrado: Internet, el boca a boca y los carteles publicitarios. Las tres formas de recibir información sobre los conciertos y festivales han sido elegidos por un 80% de las personas que han rellenado el formulario. En nuestro caso, es interesante fijarnos en el porcentaje de Internet, ya que esto nos muestra que podemos optar por ofrecer información relacionado con este tema en nuestra página web. 
+
+Siguiendo con el tema de los conciertos, también hemos preguntado si compran las entradas por Internet (todos han respondido que sí) y cómo lo hacen. El 80% de los encuestados afirma que la vía más habitual para la compra de entradas es escribir en el navegador “entradas para concierto X”. Otra de las opciones que se ha elegido en un menor porcentaje ha sido acudir a la página web del artista o grupo (20%) o a alguna página web musical que nos informa sobre la venta de entradas (20%). Al ver que existen páginas web musicales que ofrecen información sobre la venta de entradas, nos hemos dado cuenta de que nosotros también podemos ofrecer esta información. Sin embargo, al darnos cuenta de  la tendencia que hay todavía de acudir al navegador con frases como “comprar entradas para X” y comprar las entradas de esta forma, creemos que ofrecer esta información en nuestra página no es tan importante. 
```

Comment:
¿Sí pero no? ¿No pero sí? Tenéis que determinar de manera más clara si vais o no vais a ofrecer este tipo de contenido en vuestro sitio y por qué.